packaging/flavorgen/flavors/generators.go

```diff
@@ -312,6 +379,13 @@ func defaultNodeRegistrationOptions() kubeadmv1beta1.NodeRegistrationOptions {
 	}
 }
 
+func windowsNodeRegistrationOptions() kubeadmv1beta1.NodeRegistrationOptions {
+	return kubeadmv1beta1.NodeRegistrationOptions{
+		Name:             "{{ ds.meta_data.hostname }}",
```

Comment:
is this a capi thing ? what substitutes thos `{{` ? maybe a comment on this guy just to clarifiy

Reply:
it's jinja templating in cloud-init.